packages/miniflare/src/shared/dev-registry.ts

```diff
@@ -282,5 +282,8 @@ export class DevRegistry {
 }
 
 export function getDefaultDevRegistryPath() {
-	return process.env.MINIFLARE_REGISTRY_PATH ?? getGlobalWranglerConfigPath();
+	return (
+		process.env.MINIFLARE_REGISTRY_PATH ??
+		path.join(getGlobalWranglerConfigPath(), "registry")
```

Comment:
This should match wrangler settings:

https://github.com/cloudflare/workers-sdk/blob/7bde13c97287cb9d1de2ad0959ee5a6cf1db58ed/packages/wrangler/src/environment-variables/misc-variables.ts#L249-L254